utils/user_agent.jsx

```diff
@@ -57,7 +57,7 @@ export function isSafari() {
 }
 
 export function isIosSafari() {
```

Comment:
Just to check, will this mean that I can't load the desktop experience in iPad Safari? Or is it still smart enough to recognize the width and avoid showing the purely mobile view?

Reply:
This change should not affect how the app is presented. This function is only used to determine if we should show the "download app" page, and for the one case in `create_post` mentioned in the PR's description. The overall responsiveness of the app is unaffected.